src/main/include/RobotContainer.h

```diff
@@ -303,6 +334,30 @@ class RobotContainer
       1.5_mps
     };
 
+    */
+
+   //THESE ARE THE TEST POINTS FOR JUSTSHOOTCENTER OUR TEST PATH
```

Comment:
rename all our blue paths to have blue in front of it? Not a priority but could make code cleaner/easier to understand